scripts/generateAssetData/generateRelatedAssetIndex/generateRelatedAssetIndex.ts

```diff
@@ -34,6 +33,19 @@ axiosRetry(axiosInstance, { retries: 5, retryDelay: axiosRetry.exponentialDelay
 const ZERION_API_KEY = process.env.ZERION_API_KEY
 if (!ZERION_API_KEY) throw new Error('Missing Zerion API key - see readme for instructions')
 
+// Redeclared here due to ethers import issues
+export const EVM_CHAIN_IDS = [
+  KnownChainIds.EthereumMainnet,
+  KnownChainIds.AvalancheMainnet,
+  KnownChainIds.OptimismMainnet,
+  KnownChainIds.BnbSmartChainMainnet,
+  KnownChainIds.PolygonMainnet,
+  KnownChainIds.GnosisMainnet,
+  KnownChainIds.ArbitrumMainnet,
+  KnownChainIds.ArbitrumNovaMainnet,
+  KnownChainIds.BaseMainnet,
+] as const
+
```

Comment:
FYI @woodenfurniture I actually did the exact same in https://github.com/shapeshift/web/pull/7736#discussion_r1756776512 for the same reason - going to nuke all chain-adapters consumptions and leverage utils only in a follow-up 🙏🏽